data/json/items/comestibles/raw_fruit.json

```diff
@@ -472,279 +472,352 @@
     "color": "light_green",
     "spoils_in": "5 days",
     "comestible_type": "FOOD",
     "symbol": "%",
     "quench": 5,
-    "calories": 165,
-    "description": "A cluster of juicy grapes.  Quite calorie-dense compared to most fruit, and you might be able to make them into wine if you're ambitious enough.",
-    "price": 75,
+    "calories": 61,
+    "description": "A handful of juicy grapes.  Quite calorie-dense compared to most fruit, and you might be able to make them into wine if you're ambitious enough.",
+    "price": 29,
     "price_postapoc": 100,
     "material": [ "fruit" ],
     "volume": "250 ml",
     "fun": 4,
     "flags": [ "FREEZERBURN", "SMOKABLE" ],
     "smoking_result": "dry_fruit",
-    "vitamins": [ [ "vitC", 4 ], [ "calcium", 1 ], [ "iron", 2 ] ]
+    "vitamins": [ [ "vitC", 4 ], [ "calcium", 1 ], [ "iron", 1 ] ]
   },
   {
     "type": "COMESTIBLE",
     "id": "pineapple",
     "name": { "str": "pineapple" },
-    "weight": "174 g",
+    "weight": "905 g",
     "color": "brown",
     "spoils_in": "3 days",
     "comestible_type": "FOOD",
     "symbol": "%",
     "quench": 9,
-    "calories": 87,
-    "description": "A large, spiky citrus fruit.  Delicious and a great source of Vitamin C, though a bit sour.",
-    "price": 450,
+    "calories": 452,
+    "description": "A large, spiky citrus fruit.  You reckon you should not try to eat it without further preparations.",
+    "price": 765,
     "price_postapoc": 500,
     "material": [ "fruit" ],
-    "volume": "1500 ml",
-    "charges": 5,
+    "volume": "2400 ml",
     "bashing": 1,
+    "fun": -4,
+    "flags": [ "SMOKABLE" ],
+    "smoking_result": "dry_fruit",
+    "vitamins": [ [ "vitC", 480 ], [ "calcium", 9 ], [ "iron", 14 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "pineapple_chunks",
+    "name": { "str": "handful of pineapple chunks", "str_pl": "handfuls of pineapple chunks" },
+    "weight": "165 g",
+    "color": "brown",
+    "spoils_in": "3 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": 2,
+    "calories": 82,
+    "description": "A handful of pineapple chunks.  Delicious and a great source of Vitamin C, though a bit sour.",
+    "price": 139,
+    "price_postapoc": 60,
+    "material": [ "fruit" ],
+    "volume": "250 ml",
     "fun": 4,
     "flags": [ "SMOKABLE" ],
     "smoking_result": "dry_fruit",
-    "vitamins": [ [ "vitC", 93 ], [ "calcium", 2 ], [ "iron", 3 ] ]
+    "vitamins": [ [ "vitC", 87 ], [ "calcium", 1 ], [ "iron", 2 ] ]
   },
   {
     "type": "COMESTIBLE",
     "id": "coconut",
     "name": { "str": "coconut" },
-    "weight": "84 g",
+    "weight": "680 g",
     "color": "brown",
     "spoils_in": "360 days",
-    "comestible_type": "FOOD",
+    "comestible_type": "INVALID",
     "symbol": "%",
-    "quench": 2,
-    "calories": 299,
+    "quench": 10,
+    "calories": 1940,
     "description": "A tropical fruit with a hard, hairy shell and a hollow inside full of liquid.",
```

Comment:
A tropical fruit with a hard, hairy shell and a hollow inside full of liquid and edible flesh.  If you want to eat it, you'll have to find something heavy to crack open the shell and something sharp to remove the 'meat'.